proto/vpool/v1/state.proto

```diff
@@ -48,7 +50,27 @@ message Vpool {
     (gogoproto.nullable) = false
   ];
 
-  VpoolConfig config = 4 [
+  // The square root of the liquidity depth. Liquidity depth is the product of
```

Comment:
This change enforces a restart of the network, if this is not expected I would at least use a new number for new fields as protocol buffers mention.

https://learn.microsoft.com/en-us/dotnet/architecture/grpc-for-wcf-developers/protobuf-messages#field-numbers

Reply:
Yes, ideally no same field should change number. Even it is recommended when deprecating a field to leave that number set up so it cannot be taken anymore.

In this case config should keep the 4.